test/rulesets/Base/base.jl

```diff
@@ -115,6 +115,17 @@
         @test extern(dy) == extern(zeros(2, 5) .+ dy)
     end
 
+     @testset "ldexp" begin
+            rng = MersenneTwister(123456)
+            x, Δx, x̄ = 10rand(rng, 3)
+            Δz = rand(rng)
+
+            for n in (0,1,20)
+                #frule_test(ldexp, (x, Δx), (n, nothing))
```

Comment:
Can this have an issue openned on ChainRulesTestUtils.jl
and then a commented added here:
```
# TODO: the following doesn't work because of bug in ChainRulesTestUtils
# see [URL]
```